lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -189,8 +195,10 @@ def configure(conf)
         # Check for specialized GCE environments (Managed VM or Dataflow).
         # TODO: Add config options for these to allow for running outside GCE?
         attributes = fetch_gce_metadata('instance/attributes/').split
-        if attributes.include?('gae_backend_name') &&
-           attributes.include?('gae_backend_version')
+        if @force_compute_service
```

Comment:
detect_gce_subservice sounds good to me. It might be slightly pigeonholing the config options if you later want to support subservices on non-GCE platforms (as the TODO indicates). Maybe just detect_subservice or even detect_service? detect_service would match the fact that the platforms are sort of separate from the services.